packages/appstore/src/components/wallet-transfer/wallet-transfer.tsx

```diff
@@ -52,55 +51,132 @@ const WalletTransfer = observer(({ is_wallet_name_visible, setIsWalletNameVisibl
         );
     }, [from_account?.label, to_account?.label]);
 
+    const is_amount_to_input_disabled = !to_account;
+
     return (
         <div className='wallet-transfer'>
-            <div className='wallet-transfer__tile'>
-                <AmountInput
-                    currency={from_account?.currency || ''}
-                    decimal_places={from_account?.currency ? getDecimalPlaces(from_account?.currency) : 0}
-                    disabled={false}
-                    initial_value={from_amount}
-                    label={localize('Amount you send')}
-                />
-                <Divider />
-                <TransferAccountSelector
-                    is_mobile={is_mobile}
-                    is_wallet_name_visible={is_wallet_name_visible}
-                    label={localize('Transfer from')}
-                    onSelectAccount={setFromAccount}
-                    placeholder={localize('Select a trading account or a Wallet')}
-                    portal_id={portal_id}
-                    setIsWalletNameVisible={setIsWalletNameVisible}
-                    transfer_accounts={transfer_accounts}
-                    // TODO: 'Demo USD Wallet' should be replaced to the current open wallet after connecting to API call
-                    wallet_name='Demo USD Wallet'
-                    value={from_account}
-                />
-            </div>
-            <div className='wallet-transfer__tile'>
-                <AmountInput
-                    currency={to_account?.currency || ''}
-                    decimal_places={to_account?.currency ? getDecimalPlaces(from_account?.currency) : 0}
-                    disabled={!to_account}
-                    initial_value={to_amount}
-                    label={localize('Amount you receive')}
-                />
-                <Divider />
-                <TransferAccountSelector
-                    is_mobile={is_mobile}
-                    is_wallet_name_visible={is_wallet_name_visible}
-                    label={localize('Transfer to')}
-                    onSelectAccount={setToAccount}
-                    placeholder={!to_account ? localize('Select a trading account or a Wallet') : ''}
-                    portal_id={portal_id}
-                    setIsWalletNameVisible={setIsWalletNameVisible}
-                    transfer_accounts={to_account_list}
-                    transfer_hint={transfer_to_hint}
-                    // TODO: 'Demo USD Wallet' should be replaced to the current open wallet after connecting to API call
-                    wallet_name='Demo USD Wallet'
-                    value={to_account}
-                />
-            </div>
+            <Formik
+                initialValues={{
+                    to_amount: 0,
+                    from_amount: 0,
+                }}
+                onSubmit={() => {
+                    // console.log('submit');
```

Comment:
Do we need this method? or this log?

Reply:
@hamid-deriv here i added it as a plug, because if i remove onSubmit method, Formik will complain